src/Tribe/Timezones.php

```diff
@@ -164,7 +172,7 @@ protected static function get_event_timestamp( $event_id, $type = 'Start', $time
 			: $timezone;
 
 		$localized = self::to_tz( $datetime, $tzstring );
-		return strtotime( $localized );
+		return $timestamps[ $cache_key ] = strtotime( $localized );
```

Comment:
:no_entry_sign: Assignments must be the first block of code on a line

<sub>hash: 9eee0aa67144a56b5d2de6c9d3e509c7</sub>

Reply:
Ignoring.